Makefile

```
@@ -114,6 +114,10 @@ format: .bin/goimports
 docker:
 		docker build -f .docker/Dockerfile-build -t oryd/kratos:latest-sqlite .
 
+# Runs the documentation tests
+test-docs:
```

I think this should be phony and require node_modules to be installed:

```suggestion
.PHONY: test-docs
test-docs: node_modules
```